### PR TITLE
Atag Briefs

### PR DESCRIPTION
**Pull request by daniel-montalvo**

### CODE REVIEW

File: content/education.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+# NEW: Footer below has several options, and not all will be relevant for specific pages. (Ask Shawn if questions.)
+
+# In the footer below:
+# Do not translate or change CHANGELOG or ACKNOWLEDGEMENTS.
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+# Do not change the dates in the footer below.
+footer: >
+   <p><strong>Date:</strong> Updated 8 December 2022.<!-- First published December 2022. --></p>
```
Comment (daniel-montalvo) [inline content/education.md:48]:
```suggestion
   <p><strong>Date:</strong> First published 8 December 2022.</p>
```

File: content/no-code.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+# NEW: Footer below has several options, and not all will be relevant for specific pages. (Ask Shawn if questions.)
+
+# In the footer below:
+# Do not translate or change CHANGELOG or ACKNOWLEDGEMENTS.
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+# Do not change the dates in the footer below.
+footer: >
+   <p><strong>Date:</strong> Updated 8 December 2022.<!-- First published December 2022. --></p>
```
Comment (daniel-montalvo) [inline content/no-code.md:48]:
```suggestion
   <p><strong>Date:</strong> First published 8 December 2022.</p>
```

File: content/social-media.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+# NEW: Footer below has several options, and not all will be relevant for specific pages. (Ask Shawn if questions.)
+
+# In the footer below:
+# Do not translate or change CHANGELOG or ACKNOWLEDGEMENTS.
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+# Do not change the dates in the footer below.
+footer: >
+   <p><strong>Date:</strong> Updated 8 December 2022.<!-- First published December 2022. --></p>
```
Comment (daniel-montalvo) [inline content/social-media.md:47]:
```suggestion
   <p><strong>Date:</strong> First published 8 December 2022.</p>
```

Comment (shawna-slh) [reply content/social-media.md:47]: Hey Daniel. I would mildly like to leave it as is, so it's consistent with the other pages now and when/if we update it later. But only mildly, so if you want to change it, that's OK with me.

**Comment by shawna-slh** [comment]
Hey Daniel. I'm pretty sure that EOWG discussed the title for education, and decided that it should be "Standards to Make Your LMS Accessible" not Guidelines -- based on this particular audience, which is big on standards. We can dig up minutes of needed.

**Pull request merged**